bundles/io/org.eclipse.smarthome.io.voice/OSGI-INF/PocketsphinxConsoleCommandExtension.xml

```diff
@@ -1,17 +1,17 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I assume this is because you temporarily TTSServiceMacOS.xml and forgot to checkout again. You should not delete TTSServiceMacOS.xml but introduce PocketsphinxConsoleCommandExtension.xml